README.md

```diff
@@ -23,6 +23,8 @@ Getting started with the OpenHIM-core
 
 To make use of your own custom configurations you can copy the [default.json](https://github.com/jembi/openhim-core-js/blob/master/config/default.json) config file and override the default setting:
 
+* Please specify the timezone's offset value otherwise the offset used will be that of the server's timezone.  You can do this by changing the value of the utcOffset in the default.json.
```

Comment:
You know what I mean

Also could you move it down to below the copy file instructions
```suggestion
 > To change the report bluh bluh bluh change the value of `utcOffset` in `config/default.json`.
```